src/main/java/com/thealgorithms/sorts/BubbleSort.java

```diff
@@ -37,23 +37,14 @@ public <T extends Comparable<T>> T[] sort(T[] array) {
      * Driver Code
      */
     public static void main(String[] args) {
+        BubbleSort bubbleSort = new BubbleSort();
 
         Integer[] integers = {4, 23, 6, 78, 1, 54, 231, 9, 12};
-        BubbleSort bubbleSort = new BubbleSort();
         bubbleSort.sort(integers);
-
-        for (int i = 0; i < integers.length - 1; ++i) {
-            assert integers[i] <= integers[i + 1];
-        }
-        print(integers);
-        /* output: [1, 4, 6, 9, 12, 23, 54, 78, 231] */
+        print(integers); // output: [1, 4, 6, 9, 12, 23, 54, 78, 231]
 
         String[] strings = {"c", "a", "e", "b", "d"};
         bubbleSort.sort(strings);
-        for (int i = 0; i < strings.length - 1; i++) {
-            assert strings[i].compareTo(strings[i + 1]) <= 0;
-        }
-        print(bubbleSort.sort(strings));
-        /* output: [a, b, c, d, e] */
+        print(bubbleSort.sort(strings)); // output: [a, b, c, d, e]
     }
```

Comment:
Could you just delete the whole `main()`? It's redundant since tests are in place